homeassistant/components/microsoft_face.py

```diff
@@ -49,6 +49,7 @@
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.Schema({
         vol.Required(CONF_API_KEY): cv.string,
+        vol.Optional(CONF_SERVER_LOC, default="westus"): cv.string,
```

Comment:
We should not short the constant name. So use `CONF_SERVER_LOCATION`. I'm not usre if `CONF_API_ENDPOINT` a better choise for the name. I have the choise